client/layout/index.jsx

```diff
@@ -117,7 +119,9 @@ Layout = React.createClass( {
 			<MasterbarLoggedIn
 				user={ this.props.user }
 				section={ this.props.section.group }
-				sites={ this.props.sites } />
+				sites={ this.props.sites }
+				setNextLayoutFocus={ this.props.setNextLayoutFocus }
```

Comment:
A fair question. I was just trying to `connect` as few new components as possible with this change, since it can break tests. Do you think that would be a better flow here?